### PR TITLE
polyrhythmix 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by dredozubov**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

**Comment by dredozubov** [comment]
Fails with a
```
==> FAILED
Full audit polyrhythmix --online --new-formula output
  polyrhythmix
    * GitHub repository too new (<30 days old)
  Error: 1 problem in 1 formula detected.
```

Since I published the software a few days ago, the repo is new as well.

**Comment by dredozubov** [comment]
What is "notability"? What's the decision process?

**Comment by chenrui333** [comment]
> GitHub repository too new (<30 days old)

feel free to do the formula re-submission after 30days.

**Comment by chenrui333** [comment]
```
$ curl -s https://api.github.com/repos/dredozubov/polyrhythmix | jq .created_at
"2023-06-27T13:10:17Z"
```

File: Formula/polyrhythmix.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  license "Unlicense"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/poly", "-K", "4xxxx"
```
Comment (chenrui333) [inline Formula/polyrhythmix.rb:15]: maybe a better test
```
    expected = <<~EOS
      No output file path was supplied, running a dry run...
      Converges over 1 bar
    EOS
    output = shell_output("#{bin}/poly -K 4xxxx")
    assert_equal expected, output
```

also looks like the version output mismatched with the artifact

```
$ /opt/homebrew/Cellar/polyrhythmix/0.1.0/bin/poly --version
Polyrhythmix 0.1
```

Comment (dredozubov) [reply Formula/polyrhythmix.rb:15]: Good catch regarding the version, it was hardcoded in this version, I've updated it in the polyrhythmix repo itself and it should be included in the release script later.

Considering the test idea, I'm not sold on that idea, as it increases the potential for breakage with version changes.

**Pull request closed without merging**